install-scripts/helpers/check_docker-compose.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Exit values
+# 0 correct version installed
+# 3 not installed
+# 4 older than required minimum version
+# 5 newer than required maximum version
+
+if [ ! -x "$(command -v docker-compose)" ]; then
+	exit 3
+fi
+
+MIN_VERSION_MAJOR=1
+MIN_VERSION_MINOR=17
+MAX_VERSION_MAJOR=1
+MAX_VERSION_MINOR=23
```

Comment:
We'll want to document these min/max versions of docker-compose. The min is already documented, but the max is not in the README